miden-lib/src/transaction/mod.rs

```diff
@@ -161,6 +160,50 @@ impl TransactionKernel {
             .expect("Invalid stack input")
     }
 
+    /// Extends the advice inputs with account data and Merkle proofs.
+    pub fn extend_advice_inputs_for_account(
```

Comment:
The FPI tests could use this function for extending the advice inputs now, right? And so then it would be tested as well

Reply:
I think we can use it for FPI tests, but in that case `merkle_path` should become an `Option`. If it is not a problem, I can adjust this function to work with FPI.